readme.html

```diff
@@ -157,14 +280,10 @@ <h2>Set up Interactivity in Slack</h2>
 <a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_enable_interactive_messages.png">
     <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_enable_interactive_messages.png">
 </a>
-<br><br>
-In the <b>Request URL</b> text box, add the <b>POST incoming for Slack to this location</b> URL found in the <b>Asset Settings</b> window. Before saving these changes, add the username and password of the new user added to Phantom. The URL should end up in the format:
-<br><br>
-https://&lt;username&gt;:&lt;password&gt;@&lt;phantom_hostname&gt;/rest/handler/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/slack
-After adding the full URL, click <b>Enable Interactive Messages</b>
+
 <h2>Slack Bot</h2>
 <h3>Ingest Settings</h3>
-To run the Phantom SlackBot that will get Phantom to take commands from Slack, ingestion needs to be enabled on the Phantom Slack Asset. To do this go back to the INGEST SETTINGS tab and enable polling and specify the Polling interval as 10 minutes. The "Label to apply to objects from this source" setting is ignored by this app, so it can be set to anything.
+To run the Phantom SlackBot that will get Phantom to take commands from Slack, ingestion needs to be enabled on the Phantom Slack Asset. To do this go back to the INGEST SETTINGS tab and enable polling and specify the Polling interval as <b>off</b>. The "Label to apply to objects from this source" setting is ignored by this app, so it can be set to anything.
```

Comment:
Phantom or SOAR here? Did we update the Phantom terminology for the SlackBot and associated stuff?

Reply:
Updating it to SOAR everywhere in the docs

Reply:
Done